### ISSUE TITLE
panic: Failed to execute API request using Synology docker

### ISSUE THREAD

**Issue by starkayc** [gempir/justlog#103]
Hey, I read your blog and saw you were running JustLog on your Synology via docker. I attempted to do the same but encountered an error and wasn't sure how to proceed. I attempted to add a cert via synology as I read online it's a common issue but no luck. So, I was wondering how you were able run. Thanks!

Docker Settings: https://imgur.com/a/PVxJ5aB
Config: https://imgur.com/a/qcJH2ct

Log:
INFO[0000] Loaded config from /etc/justlog.json         

panic: Failed to execute API request: Post "https://id.twitch.tv/oauth2/token?client_id=removed&client_secret=removed&grant_type=client_credentials": x509: certificate signed by unknown authority

goroutine 1 [running]:

github.com/gempir/justlog/helix.NewClient(0xc0000b02e0, 0x1e, 0xc0000b0300, 0x1e, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)

	/go/src/github.com/gempir/justlog/helix/user.go:48 +0x43c

main.main()

	/go/src/github.com/gempir/justlog/main.go:27 +0x1af

**Comment by gempir** [comment]
I haven't run it in Synology forever. Will have to come back to you later on that issue. Sounds weird though.

**Comment by starkayc** [comment]
I ended up being able to fix this. I copied the certs from `/etc/ssl/certs` to `/var/packages/Docker/etc/cert` and ran the docker again and it didn't work. So, I tried running the docker on a vps and that seemed to work as intened. So, I copied the settings that I used on that onto my synology and rebooted the synology and attempted to run the docker and again and it worked. So, I'm not sure if that reboot was the fix because the certs weren't loaded or if changing the volume/mount was the fix. But, weirdly enough it loads things from `config.json` and not `justlog.json` but I guess it's because I have it that way? New to docker but yeah thought I'd update as I got it fixed :D
